canonical/store-components
# Update card designs

## Done

Changed the design of cards

## QA
- Go to https://store-components-78.demos.haus/?path=/story/bundlecard--default-story
- Check that it matches the bundle card design
- Go to https://store-components-78.demos.haus/?path=/story/charmcard--default-story
- Check that it matches the charm card design

Designs: https://app.zeplin.io/project/63cfa77260818d13bbbf1cb1/screen/63da2a4afae89a110abe8fc3

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-6709
